tests/components/yeelight/test_init.py

```diff
@@ -32,13 +40,13 @@ async def test_setup_discovery(hass: HomeAssistant):
         assert await hass.config_entries.async_setup(config_entry.entry_id)
         await hass.async_block_till_done()
 
-    assert hass.states.get(f"binary_sensor.{NAME}_nightlight") is not None
-    assert hass.states.get(f"light.{NAME}") is not None
+    assert hass.states.get(ENTITY_BINARY_SENSOR) is not None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)